[PATCH] building rquota_xdr.c depends on rquota.h

Upstream-status: Pending

rquota.h is generated automatically, and building rquota_xdr.c
needs rquota.h, so add the dependency on rquota.h for rquota_xdr.c

Signed-off-by: rongqing li <rli2@yow-blade4.wrs.com>
---
 src/plugins/quota/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/quota/Makefile.am b/src/plugins/quota/Makefile.am
index 8a6988d..3626d79 100644
--- a/src/plugins/quota/Makefile.am
+++ b/src/plugins/quota/Makefile.am
@@ -70,7 +70,7 @@ RQUOTA_XDR = rquota_xdr.c
 RQUOTA_XDR_LO = rquota_xdr.lo
 #RQUOTA_X = /usr/include/rpcsvc/rquota.x
 RQUOTA_X = $(srcdir)/rquota.x
-rquota_xdr.c: Makefile $(RQUOTA_X)
+rquota_xdr.c: Makefile $(RQUOTA_X) rquota.h
 	if [ "$(top_srcdir)" != "$(top_builddir)" ]; then \
 	  cp $(RQUOTA_X) $(top_builddir)/src/plugins/quota/; \
 	fi; \
-- 
1.8.2.1

